aws/rust-runtime/aws-sigv4/src/http_request/mod.rs

```diff
@@ -136,7 +134,7 @@ pub enum SignableBody<'a> {
 /// - x-amz-date
 /// - x-amz-content-sha-256
 /// - x-amz-security-token
-pub fn sign_core<'a, B>(
+fn sign_core<'a, B>(
```

Comment:
this function is used externally, I expect a future change will take care of it

Reply:
Yeah, one of my later commits in this PR renames it and makes it public again.